creator-node/src/routes/files.js

```diff
@@ -142,15 +142,17 @@ const getCID = async (req, res) => {
     )
```

Comment:
Worth probably updating the comment of the top of CID.

how's

```
/**
 * Given a CID, return the appropriate file.
 * 1. Try to stream the file directly from the filesystem
 * 2. Ensure that the CID exists in this node's database, then try to fetch the file from
 *      the content node network, save it, and stream from the filesystem
 * 3. Ensure that the CID exists in this node's database, then fall
 *      back to trying to find the file from IPFS 
 * If none of the above work, respond with a server error.
 */
```

Reply:
I wrote this comment to ensure my understanding too

Reply:
yep updating